pkg/util/fs/proc/proc.go

```diff
@@ -144,7 +144,10 @@ func ReadIDMap(path string) (uint32, uint32, error) {
 	defer r.Close()
 
 	scanner := bufio.NewScanner(r)
-	scanner.Scan()
+	tok := scanner.Scan()
+	if !tok {
+		return 0, 0, scanner.Err()
```

Comment:
Btw error can be nil here if the file was empty. I am not sure how that will affect further logic since it may become `0`  for uid, so better ask @cclerget.

Reply:
`/proc/self/uid_map` or `/proc/self/gid_map` are never empty, but just in case maybe wrap the error with something `fmt.Errorf("scanner error: %s", scanner.Err())` to be sure

Reply:
hm, that might be confusing, `scanner error: nil`
but assuming that files are never empty...